internal/core/entities/contact.go

```diff
@@ -1,13 +1,15 @@
 package entities
 
-import "github.com/google/uuid"
+import (
+	"github.com/google/uuid"
+)
 
 type Contact struct {
 	DBModel
 	UserID      uuid.UUID `json:"user_id"`
 	PhoneBookID uuid.UUID `json:"phone_book_id"`
 	FirstName   string    `gorm:"size:255" json:"first_name"`
 	LastName    string    `gorm:"size:255" json:"last_name"`
-	Username    string    `gorm:"size:255;" json:"user_name"` // Has to be unique based on phonebook
+	Username    string    `gorm:"size:255;unique" json:"Username"`
```

Comment:
json should be lowercase, either "username" or "user_name" to match the "first_name" and "last_name"

Reply:
Yes, you are right. I fixed it.